### ISSUE TITLE
export --help-type "complains" instead of providing list of options

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#1039]
I guess it was intended (the same on 0.3.1 and last commit which modified datalad/export f2e48aaeb9730deec5f06d27e743fc5f282e169f , so probably never worked) to output summary of options for that backend but it just exits with error code

``` bash
(git)hopa:~datalad/datalad[enh-save]git
$> datalad export --help-type
usage: datalad export [--version] [-h] [-l LEVEL] [-p {condor}] [-d DATASET]
                      [--help-type] [-o OUTPUT]
                      {tarball}
datalad export: error: too few arguments
(venv-tests) 2 14040 ->2
```

**Comment by mih** [comment]
You did not provide an exporter to get help for. The list of supported exporters is in the output you posted.

````
% datalad export tarball --help-type  
Just call it, and it will produce a tarball.
````

Current documentation for `--help-type` is:

> show help for a specific export type/format.

I think this is OK. Will close this one. Please suggest what should change, if you disagree.

**Comment by yarikoptic** [comment]
Aha... So the tarball argument is mandatory... I just followed output of --help and didn't realize that the last piece (atm a single one) is mandatory. Friendly message would say that I had to choose 1 out of 1 known plugins and surviving it in the command line

**Comment by mih** [comment]
Fixed in #1619
